spec/tracebacks_spec.lua

```diff
@@ -3,14 +3,15 @@
 -- Please refer to the LICENSE and AUTHORS files for details
 -- SPDX-License-Identifier: MIT
 
-local util = require "misc.util"
+local util = require "spec.util"
```

Comment:
Now that `util.lua` is out of the way, can we expect the `pt-run` source code to be in `src` directory as a single file, e.g. `pt-run.c` or `main.c`?

Reply:
I'm working on a PR to fix the makefile and will decide based on what makes things easier for that. Currently, src/pt-run/main.c --> src/pt-run.c  and lib/ptracer.h --> src/ptracer.h. But that's not set in stone yet.